console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx

```diff
@@ -53,7 +53,7 @@ const FinishedStep: React.FunctionComponent<IFinishedStepProps> = ({
     if (routeDetail && projectType === ProjectType.IOT_PROJECT) {
```

Comment:
can you try to destructure routeDetail and avoid to call it gain.
const {name,namespace,type}=routeDetail;